internal/config/config.go

```diff
@@ -12,10 +12,9 @@ import (
 
 const (
 	LinkerdOperation = "linkerd"
-	Development      = "development"
```

Comment:
We should talk to @kumarabd, let him confirm this change can be true

Reply:
@Aisuko Actually I made this change in accordance with the previous discussion where we concluded that the running application shall stay oblivious of its running environment :sweat_smile: